tests/vagrant/test.yml

```diff
@@ -13,7 +13,7 @@
         - build
 
   roles:
-    - name: ansible-city.nginx
+    - role: ansible-city.nginx
```

Comment:
Could you change ansible-city to sansible? ansible-city is an old version of this org, there will probably be other references in the makefile as well.